callback_plugins/custom_logger.py

```diff
@@ -63,18 +66,41 @@ def log_task_result(self, host, result, task_name):
             self.results[host][result] += 1
 
     def log_summary_results(self, host):
```

Comment:
I think that some of the issues I saw are addressed with L75-77, since __some__ of the issues I saw were related to when the host didn't have any results (e.g. if there were no tasks with the keywords run on that host)